open_discussions/views.py

```diff
@@ -47,7 +49,8 @@ def _render_app(request, initial_state=None):  # pylint:disable=unused-argument
         user_email = user.email
         user_is_superuser = user.is_superuser
         user_id = user.id
-        user_list_staff = is_list_staff(user)
+        user_public_list_editor = is_public_list_editor(user)
```

Comment:
I changed this variable name here (and in the front end code it's passed to) because I felt the original name would be too confusing now that we are adding staff lists.